templates/analytics/realm_summary_table.html

```diff
@@ -50,6 +63,14 @@ <h2 style="text-align: center;">{{ num_active_sites }} active sites</h2>
                 {{ row.date_created_day }}
             </td>
 
+            <td class="envelope">
+                {% if not loop.last %}
```

Comment:
Why do we need this conditional here?

Reply:
Last row in table is used for `Total`, So, there is no admin for that. Therefore, I used conditional statement.